scripts/linux/runE2ETest.sh

```diff
@@ -1118,6 +1119,12 @@ if [[ "${TEST_NAME,,}" == "longhaul" ]]; then
     TWIN_UPDATE_SIZE="${TWIN_UPDATE_SIZE:-1}"
     TWIN_UPDATE_FREQUENCY="${TWIN_UPDATE_FREQUENCY:-00:00:15}"
     TEST_START_DELAY="${TEST_START_DELAY:-00:00:00}"
+
+    # if Run log level is debug, we need ~96M for 24hr of edgeHub/edgeAgent log
+    if [[ "${RUNTIME_LOG_LEVEL,,}" == "debug" ]]; then
```

Comment:
I think you want to use different log file size for different architecture, rather than log level.  I don't agree that we can only use 4m x 7 on Linux ARM32 device.  And why 96M for AMD64? did you check the average LH log file size per day? and why we don't turn on debug log level for Linux arm32?